volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -790,12 +793,29 @@ def get_sids(self) -> Iterable[str]:
 
 
     def privileges(self):
-        "Return a list of privileges for the current token object."
-        for priv_index in range(64):
-            yield (priv_index,
-                   bool(self.Privileges.Present & (2**priv_index)),
-                   bool(self.Privileges.Enabled & (2**priv_index)),
-                   bool(self.Privileges.EnabledByDefault & (2**priv_index)))
+        """Return a list of privileges for the current token object."""
+        
+        try:
+            for priv_index in range(64):
+                yield (priv_index,
+                       bool(self.Privileges.Present & (2**priv_index)),
+                       bool(self.Privileges.Enabled & (2**priv_index)),
+                       bool(self.Privileges.EnabledByDefault & (2**priv_index)))
+        except AttributeError: # Windows XP
+            layer_name = self.vol.layer_name
+            kvo = self._context.layers[layer_name].config["kernel_virtual_offset"]        
+            symbol_table = self.get_symbol_table_name()
+            ntkrnlmp = self._context.module(symbol_table,
+                                            layer_name = layer_name,
+                                            offset = kvo)
+            if self.PrivilegeCount < 1024:
```

Comment:
What happens if this isn't the case?  It looks like we'd return an empty list.  If we'd gone to the effort of differentiating this, it seems like we should log when it goes wrong?